internal/service/privatelinkendpointservicedatafederationonlinearchive/resource_privatelink_endpoint_service_data_federation_online_archive.go

```diff
@@ -53,6 +54,17 @@ func Resource() *schema.Resource {
 				Optional: true,
 				ForceNew: true,
 			},
+			"region": {
+				Type:             schema.TypeString,
+				Optional:         true,
+				ForceNew:         true,
```

Comment:
Using `ForceNew` here since the Update function is not implemented, but I'm uncertain if it is correct.
On this resource, I think only `comment` should be `ForceNew: false`, (but it is currently `true`).
The [api](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Data-Federation/operation/createDataFederationPrivateEndpoint)  don't expose a `PUT` endpoint but mentions updating when only `comment` has changed.

Is it worth it to implement the Update function?

Reply:
I would just try this out via the API once and if updating only comment is supported correctly (i.e. no impact to remaining attributes) then I don't see why we cannot support update for comment